Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1206,6 +1206,112 @@ String HTMLInputElement::covert_number_to_string(double input) const
     VERIFY_NOT_REACHED();
 }
 
+// https://html.spec.whatwg.org/multipage/input.html#attr-input-min
+Optional<double> HTMLInputElement::min() const
+{
+    // If the element has a min attribute, and the result of applying the algorithm to convert a string to a number to the value of the min attribute is a number, then that number is the element's minimum; otherwise, if the type attribute's current state defines a default minimum, then that is the minimum; otherwise, the element has no minimum.
```

Comment:
Please break very long lines into ~120 characters per line (here and elsewhere in this commit)